### DISCUSSION TITLE
Text rescaling not working

### DISCUSSION THREAD

**Discussion by locutorprime** [Open-Shell/Open-Shell-Menu#765] [Q&A]
I’m a long-time user of Classic Shell.  I just got a new 4K monitor and found that my Start menus weren't observing the change when I rescale to 175%.  I installed Open Shell and it rescaled my Start menu properly… for a while.  But it stopped rescaling and the menu is now too tiny to read.  "Override system DPI" is set to zero, so it should be upscaling to 175% like everything else but isn’t.  Suggestions?

**Comment by bonzibudd** [comment]
I think that Microsoft may have changed the API for Win32, or some other bug is causing the default text to not resize all of the way. What skin are you using for the Start menu?

**Reply by locutorprime** [reply]
Thanks for the reply.  I'm using the classic skin.  Just to update: a reboot of my PC a few days ago restored the proper DPI, and so far it’s staying there (knock on wood).

**Comment by Diwax** [comment]
I don't know what skin I'm using... btw it's not the resizing that is the is issue, it that it is popping up every time I turn on the computer. I just finally uninstalled 'Shell' and see if it affects anything... otherwise I do a restore or reinstall it.

________________________________
